app/src/main/java/com/alexchan/wallpaper/ui/search/searchActivity/SearchActivity.kt

```diff
@@ -0,0 +1,87 @@
+package com.alexchan.wallpaper.ui.search.searchActivity
```

Comment:
Don't think you need another activity since you are using Navigation component

Reply:
According to the screenshot for Search Activity in https://github.com/hua9495/wallpaper-android/pull/6#issuecomment-656027048, with the implementation of recent suggestion and replacing the action bar with search view and as action view, it is much easier to handle by using a new activity, but if it's not a good design and coding practice, I can try to change and implement it using Fragment.